{"name":"Review repository find by id failure","fullName":"TestReviewFindByIDSuite/Review repository find by id/TestFindByID_Failure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727773563256,"stop":1727773563256,"uuid":"62682084-7fd4-11ef-a987-ca15b1947338","historyId":"3ddab65d290296d1debfe4aa28c19359","testCaseId":"d678d2f53e249a9002a6d3750ab79dd8","labels":[{"name":"language","value":"go1.22.5"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"h19.net58.bmstu.ru"},{"name":"thread","value":"TestReviewFindByIDSuite/Review repository find by id/TestFindByID_Failure"},{"name":"suite","value":"Review repository find by id"},{"name":"package","value":"github.com/paw1a/eschool/internal/adapter/repository/postgres/test"}],"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1727773563256,"stop":1727773563256,"parameters":[{"name":"Error","value":"sql: no rows in result set: record does not exist"},{"name":"Target","value":"record does not exist"}]}]}